src/datatype.ts

```diff
@@ -12,6 +12,20 @@ import {ISelectAble, SelectAble} from './idtype';
 import {all, none, Range1D, Range1DGroup, composite, Range} from './range';
 
 /**
+ * TODO
+ */
+export interface IDataDescriptionValue {
```

Comment:
in either case this interface is just valid for `int` and `real` value types. `categorical` ones doesn't have range